libs/components/indicators/src/lib/modules/icon/icon.component.ts

```diff
@@ -29,21 +29,21 @@ export class SkyIconComponent {
   public iconType: SkyIconType | undefined;
 
   /**
-   * Specifies the size of the icon using
+   * The size of the icon using
    * [Font Awesome sizes](https://fontawesome.com/how-to-use/on-the-web/styling/sizing-icons). Do not prefix the size with `fa-`.
    */
   @Input()
   public size: string | undefined;
 
   /**
-   * Indicates whether the icon has a fixed width.
+   * Whether the icon has a fixed width.
    * @default false
    */
   @Input()
   public fixedWidth: boolean | undefined;
 
   /**
-   * Specifies the icon variant (`"line"` or `"solid"`). If the variant doesn't exist for the
+   * The icon variant (`"line"` or `"solid"`). If the variant doesn't exist for the
```

Comment:
```suggestion
   * The icon variant. If the variant doesn't exist for the
```
The options are listed in the Type aliases section.